### PR TITLE
Added "getBuffer" method and updated certificates

### PR DESCRIPTION
**Pull request by JBtje**
form-data works with many packages, however it does not work out of the box with Axios. Many times the question has been asked as to how to make it work, but no proper solution is provided. I think it would be nice if form-data allows users to obtain the datapackage (as buffer) so they can do with it whatever they like, e.g. pass it over to Axios.

Added method: getBuffer()
Added Test: to test GetBuffer()
Updated: Certificates where signed with small key and needed replacement to make the test pass.

### CODE REVIEW

**Comment by mapleeit** [comment]
Great work. Please take a look at this! @alexindigo 

BTW. It seems CI was broken. #424 only change some docs, but it failed anyway.

File: test/fixture/cert.pem [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
-MIIBnDCCAUYCCQCgIldFYidw0zANBgkqhkiG9w0BAQUFADBVMQswCQYDVQQGEwJV
-UzELMAkGA1UECBMCQ0ExEjAQBgNVBAcTCVBhbG8gQWx0bzERMA8GA1UEChMIRm9y
-bURhdGExEjAQBgNVBAMTCWxvY2FsaG9zdDAeFw0xNTA2MTMxMzA3MTBaFw00MjEw
-MjgxMzA3MTBaMFUxCzAJBgNVBAYTAlVTMQswCQYDVQQIEwJDQTESMBAGA1UEBxMJ
-UGFsbyBBbHRvMREwDwYDVQQKEwhGb3JtRGF0YTESMBAGA1UEAxMJbG9jYWxob3N0
-MFwwDQYJKoZIhvcNAQEBBQADSwAwSAJBAOkE8JJzLQbVEo/vRVRqiwNEbiuEooJA
-099h1IJLtsPJVphSDBTtE0FLB0YIzT7bG44Drm/975XEZjoQ9TmYbTMCAwEAATAN
-BgkqhkiG9w0BAQUFAANBALSDmvKXb4cE622OdSRrDsnr8xUosetsAGSlGauWBKgc
-DNQhP6wSQHTAJT8wiVgSPYwnhcnzkZlwzdfMHMtzFIc=
+MIIFfzCCA2egAwIBAgIJALeHtgIY/UKcMA0GCSqGSIb3DQEBCwUAMFUxCzAJBgNV
```
Comment (alexindigo) [inline test/fixture/cert.pem]: Why would we need to change test certs?

**Comment by alexindigo** [comment]
Thanks @JBtje looks interesting.

@mapleeit Agreed, looks nice, not sure about test cert though.
And I'll check why Windows is unhappy again :)

File: test/fixture/cert.pem (hunk 1 above)
Comment (JBtje) [reply test/fixture/cert.pem]: With the installed certificate, I was unable to run all the tests: the certificate format was outdated (if i recall correctly). So I added a new one, all tests succeeded and pushed.

Comment (alexindigo) [reply test/fixture/cert.pem]: Thanks. I restarted windows builds 🤞

**Comment by JBtje** [comment]
It is an honour to have contributed to such an important project. Many thanks for merging!

**Comment by alexindigo** [comment]
@JBtje Thanks for your help and pushing it forward. I'll publish your change shortly.

**Comment by alexindigo** [comment]
Published as part of `2.4.0` – https://www.npmjs.com/package/form-data
Thank you.

**Comment by mapleeit** [comment]
Good to know that. Thanks!

**Comment by mapleeit** [comment]
In addition, I'm wondering is there any document for all APIs? I couldn't find that in README or anywhere else.

**Comment by JBtje** [comment]
I don't think there is... I realised it yesterday that my merge req. didn't have documentation update. But havent had the time to make a proper one and submit it. Will see if I can do that upcomming weekend.

**Comment by mapleeit** [comment]
Sounds great. If you need any help, please contact me. : )

**Comment by alexindigo** [comment]
Thank you for doing it @mapleeit

**Pull request merged**